benchmarks/Gemfile

```diff
@@ -6,16 +6,27 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby ">= 2.7.7"
 
-gem "benchmark-ips", "2.13.0"
+gem "benchmark", "0.3.0"
```

Comment:
I might've missed it below—what do we need this for?

Reply:
At some point I was getting a failure that made me think this would help, and the failure went away, but it may not have been relevant at all.

Just tried, and it works without it, and here are no direct references.  Removing.